src/name_bazaar/ui/subs.cljs

```diff
@@ -96,3 +97,15 @@
       [(subscribe [:registrar.transfer/tx-pending? label])]
       [(subscribe [:ens.set-owner/tx-pending? node])]))
   first)
+
+
+;; TODO: move into district.ui.mobile repo
+(reg-sub
+  ::mobile-subs/coinbase-appstore-link
+  :<- [::mobile-subs/android?]
+  :<- [::mobile-subs/ios?]
+  (fn [[android? ios?]]
+    (cond
+      (true? android?) (:android-mobile-link constants/coinbase)
```

Comment:
Why not dispatch cond on just the value which is trythy/falsy?

Reply:
true! i'll fix this in the next push